app-info/Makefile

```diff
@@ -0,0 +1,48 @@
+NULL :=
```

Comment:
New files, such as this Makefile, could do with an SPDX copyright/licensing header

Reply:
:+1:

Reply:
Just go ahead and merge once this nitpick is fixed